web/components/roomConfig/GithubRepoConfig.tsx

```diff
@@ -52,29 +78,38 @@ const ConnectionSearch: FunctionComponent<{api: BridgeAPI, onPicked: (state: Git
         }
         // Things break if we depend on the thing we are clearing.
         // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [searchFn, filter, isConnected]);
+    }, [searchFn, filter]);
 
-    useEffect(() => {
-        const hasResult = results?.find(n => n.name === filter);
-        if (hasResult) {
-            onPicked(hasResult.state);
-            setCurrentRepo(hasResult.name);
-        }
-    }, [onPicked, results, filter]);
+    const onOrgPicked = useCallback((evt: InputEvent) => {
+        // Reset the search string.
+        setFilter({
+            orgName: (evt.target as HTMLSelectElement).selectedOptions[0].value,
+        });
+    }, []);
+
+    const orgListResults = useMemo(
+        () => orgs?.map(i => <option key={i.name}>{i.name}</option>),
+        [orgs]
+    );
 
     const repoListResults = useMemo(
-        () => results?.map(i => <option repo={i.state.repo} org={i.state.org} value={i.name} key={i.name} />),
+        () => results?.map(i => <option key={i.name} value={i.name} />),
         [results]
     );
 
 
     return <div>
         {isConnected === null && <p> Loading GitHub connection. </p>}
         {isConnected === false && <p> You are not logged into GitHub. </p>}
+        {isConnected === true && orgs?.length === 0 && <p> You do not have access to any GitHub organizations. </p>}
```

Comment:
Should we link to the docs on how to auth, or should that be left up to the admin/user to discover?

Reply:
May as well make the most of our docs.  However, I'll do it in a separate change to cover Jira & GitLab as well.

Reply:
Tracked: https://github.com/matrix-org/matrix-hookshot/issues/510